martin-mbtiles/Cargo.toml

```diff
@@ -21,6 +21,8 @@ serde_json.workspace = true
 sqlx.workspace = true
 thiserror.workspace = true
 tilejson.workspace = true
+tokio = { version = "1.25.0", features = ["full"] }
+anyhow = "1.0"
```

Comment:
These should be moved [here](https://github.com/maplibre/martin/blob/main/Cargo.toml#L87) (keep that list sorted), and here you just want to use `.workspace = true`. Also, move these two lines down to # Bin dependencies right below.

Using workspace deps allow all dependency versions to be managed in one place.